full-stack-engineer/submissions/michaelyao/client/src/Api.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = process.env.REACT_APP_BACKEND_URL;
+axios.defaults.withCredentials = true;
+
+export const signup = ({ username, password }) =>
+  axios.post('/auth/signup/', {
+    username,
+    password,
+  });
+
+export const signin = ({ username, password }) =>
+  axios.post('/auth/signin/', {
+    username,
+    password,
+  });
+
+export const signout = () => axios.post('/auth/signout/');
+
+export const currentUser = () => axios.get('/auth/currentUser/');
+
+export const listProducts = () => axios.get('/products/');
+
+export const listReviews = (productId) =>
+  axios.get(`/products/${productId}/reviews`);
+
+export const createReview = (productId, rating, text) =>
```

Comment:
I would have done the second parameter as a object: `(productId, { rating, text })`

Reply:
Updated